### PR TITLE
Added new python ques

### PR DESCRIPTION
**Pull request by Amit-TheOne**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

Please check my PR.

### CODE REVIEW

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -1,24 +1,4 @@
...
-  },
-  {
-    Question: "What kind of programming can JavaScript be used for?",
-    Answer: "Both Client and Server side programming",
-    Distractor1: "Server side only",
-    Distractor2: "Client side only",
-    Distractor3: "None of these options",
-    Explanation:
-      "JavaScript is a programming language which can be used on the client and server side.",
-    Link: "https://developer.mozilla.org/en-US/docs/Learn/Server-side/First_steps/Introduction#are_server-side_and_client-side_programming_the_same"
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:20]: Why are these questions deleted? Please do not remove questions unless a maintainer has approved those changes.

File: src/data/python-quiz.ts [hunk 2]
```diff
@@ -1,25 +1,13 @@
...
-   {
-    Question: 
-      "In Python, what is the output of the following code: print(list(range(3, 6)))?",
-    Answer: "[3, 4, 5]",
-    Distractor1: "[3, 4, 5, 6]",
-    Distractor2: "[3, 6]",
-    Distractor3: "[3, 3, 3, 3, 3, 3]",
-    Explanation: 
-      "The list(range(3, 6)) function returns a list of numbers from 3, increments by 1 (by default), and stops at 5 (excluding 6).",
-    Link: "https://www.freecodecamp.org/news/python-range-function-explained-with-code-examples/",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:11]: Why is this question deleted? Please do not remove questions unless a maintainer has approved those changes.

File: src/data/python-quiz.ts [hunk 3]
```diff
@@ -1581,35 +1569,39 @@ const pythonQuiz = [
...
   },
   {
-    Question: "In Python, which method is used to trim trailing whitespace from a string?",
-    Answer: "strip()",
-    Distractor1: "trim()",
-    Distractor2: "remove()",
-    Distractor3: "removesapce()",
-    Explanation: "Python has three built-in methods for trimming leading and trailing whitespace and characters from strings: strip(), lstrip(), rstrip()",
-    Link: "https://www.freecodecamp.org/news/python-strip-how-to-trim-a-string-or-line/",
-  },
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1612]: Why is this question deleted? Please do not remove questions unless a maintainer has approved those changes.

File: src/data/python-quiz.ts [hunk 4]
```diff
@@ -1581,35 +1569,39 @@ const pythonQuiz = [
...
+    Question: "In Python, clear() method in list is used to?",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1596]:
```suggestion
    Question: "In Python, what does the clear() method in a list do?",
```

File: src/data/python-quiz.ts [hunk 5]
```diff
@@ -1581,35 +1569,39 @@ const pythonQuiz = [
...
+    Answer: "To delete all the elements of the list",
+    Distractor1: "To delete the list",
+    Distractor2: "To delete the last element of the list",
+    Distractor3: "To delete the specific element of the list",
+    Explanation:
+      "In Python, clear() method in list is used to delete all the elements of the list. The list name you give inside the clear() method will be delete all the element of the list and when list is printed, the output will be the empty list.",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1602]:
```suggestion
      "In Python, the clear() method in list is used to delete all the elements of the list.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/javascript-quiz.ts (hunk 1 above)
Comment (Amit-TheOne) [reply src/data/javascript-quiz.ts]: It was mistakenly done due to a branch change.
Changes have been reverted.

File: src/data/python-quiz.ts (hunk 2 above)
Comment (Amit-TheOne) [reply src/data/python-quiz.ts:11]: It was mistakenly done due to a branch change.
Changes have been reverted.

File: src/data/python-quiz.ts (hunk 4 above)
Comment (Amit-TheOne) [reply src/data/python-quiz.ts]: Done ✅

File: src/data/python-quiz.ts (hunk 5 above)
Comment (Amit-TheOne) [reply src/data/python-quiz.ts]: Done ✅

**Comment by Amit-TheOne** [comment]
Hey @jdwilkin4 !
Please review the changes and fixes. 
Thanks!

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by Amit-TheOne** [comment]
Thank you so much!✨

**Pull request merged**